src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -600,6 +601,89 @@ MStatus PRTModifierAction::updateUI(const MObject& node) {
 	return MStatus::kSuccess;
 }
 
+MStatus PRTModifierAction::updateDynamicEnums() {
+	for (auto& e : mEnums) {
+		if (e.mValuesAttr.length() > 0) {
+
+			const MString fullAttrName = e.mAttr.name();
+			const RuleAttribute ruleAttr = reverseLookupAttribute(fullAttrName.asWChar(), mRuleAttributes);

```

Comment:
We probably should handle the error (?) case where `reverseLookupAttributes` returns `{}`.
Or even better, maybe refactor/avoid this lookup so this case cannot happen in the first place (within reasonable effort).

Reply:
Refactored to use std::map for getting attributes